docs/spec.md

```diff
@@ -6312,6 +6331,24 @@ function returns `true`. If `x` is not a tensor, returns `None`.
 tensors and returns `num_results` slices of `x` along the axis `axis`.
 If `x` is not a tensor or `dim(x, axis) % num_results != 0`, returns `None`.
 
+* `convert_with_quantized_type(x: Value, destination_type: Type)
```

Comment:
Since `destination_type` doesn't have to be quantized, let's think of a more precise name, e.g. `convert_or_quantize`.

Reply:
make sense.